modules/apps/change-tracking/change-tracking-rest-impl/src/main/java/com/liferay/change/tracking/rest/internal/resource/v1_0/ProcessResourceImpl.java

```diff
@@ -67,6 +80,20 @@ public Process getProcess(Long processId) throws Exception {
 		return new Process();
 	}
 
+	@Override
+	public Page<Process> getProcessesPage(
+			Long companyId, String keywords, ProcessType type, Long userId,
+			Pagination pagination, Sort[] sorts)
+		throws Exception {
+
+		List<Process> processes = _getProcesses(
+			_getCTProcesses(
+				companyId, GetterUtil.getLong(userId), keywords, type,
+				pagination, sorts));
+
+		return Page.of(processes, pagination, processes.size());
```

Comment:
If you don't need to return the requested pagination (the last parameter of Page.of is the totalCount), there is a method lile Page.of(processes)